Assets/Scripts/KuriTextManager.cs

```diff
@@ -35,6 +38,8 @@ public enum PRIORITY {
         Queue<TextCommand> highPriorityCommands;
         AudioSource audioSource;
         AudioClip computerNoiseClip;
+        public bool isGlowing = true;
+        public Material led_on, led_off;
```

Comment:
keep with cameral case, also these variables should stand on their own so name them `ledOnMaterial` and `ledOffMaterial`; also these should be loaded via resource loading. Look at `ResourcePathConstants.cs` (or something like that) and load it, look at this hackmd on how to do so: https://hackmd.io/RwRsuOjSTY-wTtRlbrvHeA?view

Reply:
also you will want to avoid public dragging of objects, if that doesn't make sense let me know and I can show you and Karen better ways to do it